deploy/helm-chart/templates/deployment.yaml

```diff
@@ -143,6 +143,10 @@ spec:
       securityContext:
         fsGroup: 1001
         runAsNonRoot: true
+      {{- with .Values.tolerations -}}
```

Comment:
I'm getting the following error when tolerations are defined:
```
Error: INSTALLATION FAILED: YAML parse error on nginx-gateway-fabric/templates/deployment.yaml: error converting YAML to JSON: yaml: line 114: mapping values are not allowed in this context
helm.go:84: [debug] error converting YAML to JSON: yaml: line 114: mapping values are not allowed in this context
```

This fixed it for me:
```
{{- if .Values.tolerations }}
tolerations:
{{- toYaml .Values.tolerations | nindent 6 }}
{{- end }}
```

I also changed the indent to 6 to be consistent with the rest of the template.

Reply:
I've updated as requested, though `nindent 8` would be more consistent with the rest of the template (see `affinity` and `labels`).

Reply:
Wtih `nindent 8`, the list spacing is inconsistent with the other lists in the yam: 

```
      tolerations:
        - effect: NoSchedule
          key: key1
          operator: Equal
          value: value1
      volumes:
      - name: nginx-conf
        emptyDir: {}
      - name: nginx-secrets
        emptyDir: {}
      - name: nginx-run
        emptyDir: {}
      - name: nginx-cache
        emptyDir: {}
      - name: nginx-lib
        emptyDir: {}
```